lib/std/ascii.zig

```diff
@@ -188,73 +225,103 @@ fn inTable(c: u8, t: tIndex) bool {
     return (combinedTable[c] & (@as(u8, 1) << @enumToInt(t))) != 0;
 }
 
-pub fn isAlNum(c: u8) bool {
+// remove all decls marked as DEPRECATED after 0.10.0
+
+/// DEPRECATED: use `isAlphanumeric`
+pub const isAlNum = isAlphanumeric;
+/// DEPRECATED: use `isAlpha`
+pub const isAlpha = isAlphabetic;
+/// DEPRECATED: use `isAlpha`
+pub const isCntrl = isControl;
+/// DEPRECATED: use `isWhitespace`.
+pub const isSpace = isWhitespace;
+/// DEPRECATED: use `whitespace`.
+pub const spaces = whitespace;
+/// DEPRECATED: use `isHex`.
+pub const isXDigit = isHex;
+
+/// Returns whether the character is alphanumeric. This is case-insensitive.
+pub fn isAlphanumeric(c: u8) bool {
     return (combinedTable[c] & ((@as(u8, 1) << @enumToInt(tIndex.Alpha)) |
         @as(u8, 1) << @enumToInt(tIndex.Digit))) != 0;
 }
 
-pub fn isAlpha(c: u8) bool {
+/// Returns whether the character is alphabetic. This is case-insensitive.
+pub fn isAlphabetic(c: u8) bool {
     return inTable(c, tIndex.Alpha);
 }
 
-pub fn isCntrl(c: u8) bool {
-    return c < 0x20 or c == 127; //DEL
+/// Returns whether the character is a control character.
+///
+/// See also: `control_code`.
+pub fn isControl(c: u8) bool {
+    return c <= control_code.us or c == control_code.del;
 }
 
 pub fn isDigit(c: u8) bool {
     return inTable(c, tIndex.Digit);
 }
 
+/// DEPRECATED: use `isPrint(c) and c != ' '` instead
 pub fn isGraph(c: u8) bool {
     return inTable(c, tIndex.Graph);
 }
 
+/// Returns whether the character is lowercased.
 pub fn isLower(c: u8) bool {
     return inTable(c, tIndex.Lower);
 }
 
+/// Returns whether the character has some graphical representation and can be printed.
+/// This also returns `true` for the space character.
 pub fn isPrint(c: u8) bool {
```

Comment:
Might be helpful to mention that this is just `!isControl(c)`

Reply:
Plus check for whitespace `c == ' '`.

Reply:
Great point!